modules/campaignion_share_page/campaignion_share_page.features.field_base.inc

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * @file
+ * campaignion_share_page.features.field_base.inc
+ */
+
+/**
+ * Implements hook_field_default_field_bases().
+ */
+function campaignion_share_page_field_default_field_bases() {
```

Comment:
I guess this redefines field_base that are already defined in `campaignion_thank_you_page_field_default_field_bases()`. There should only be one module that defines these and all modules with field_instances should depend on it.